packages/terra-date-input/src/DateInputField.jsx

```diff
@@ -234,18 +245,29 @@ const DateInputField = (props) => {
         disabled={disabled}
         isInvalid={isInvalid}
         isIncomplete={isIncomplete}
+        help={help}
         required={required}
-        monthAttributes={{ ...monthAttributes, ...{ 'aria-describedby': monthAriaDesciptionIds } }}
-        dayAttributes={{ ...dayAttributes, ...{ 'aria-describedby': dayAriaDesciptionIds } }}
-        yearAttributes={{ ...yearAttributes, ...{ 'aria-describedby': yearAriaDesciptionIds } }}
+        monthAttributes={{ ...monthAttributes, ...{ 'aria-describedby': monthAriaDescriptionIds } }}
+        dayAttributes={{ ...dayAttributes, ...{ 'aria-describedby': dayAriaDescriptionIds } }}
+        yearAttributes={{ ...yearAttributes, ...{ 'aria-describedby': yearAriaDescriptionIds } }}
       />
       {isInvalid && error && <div id={errorAriaDescriptionId} className={cx('error-text')}>{error}</div>}
-      {help && <div id={helpAriaDescriptionId} className={cx('help-text')}>{help}</div>}
+      {help ? (
+        <div id={helpAriaDescriptionId} className={cx('help-text')} aria-label={`Format: ${format}, ${help}`}>
```

Comment:
Should the aria-label here (`Format:`) be consistent with the aria-label in DateInput (`Date Format:`)? This label should probably be translated.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1230/commits/34514197b3cf2dec580bd2b2d8374d6809fc9d23

Reply:
Should we translate "Date Format"? Otherwise, it would be announce in english for other languages.

Reply:
Raised [jira](https://jira.cerner.com/browse/TRANS-9214) for translation.